packages/react-hooks/package.json

```diff
@@ -15,12 +15,7 @@
   "license": "MIT",
   "scripts": {
     "build": "just-scripts build",
-    "lint": "just-scripts lint",
```

Comment:
nx inferred targets are now enabled, keeping only specific tasks like build which uses v8 just-scripts logic to not create any artifact regressions